tailnet/coordinator.go

```diff
@@ -230,7 +239,9 @@ func (c *coordinator) ServeClient(conn net.Conn, id uuid.UUID, agent uuid.UUID)
 		LastWrite: now,
 	}
 	c.mutex.Unlock()
+	logger.Debug(context.TODO(), "added tracked connection")
 	defer func() {
+		logger.Debug(context.TODO(), "deleting tracked connection")
```

Comment:
Log messages are a big mixed here as in added (did)/deleting (doing). For the agent we're also just logging "did" (added/deleted). Not sure if there's a reason to do one over the other but if there's not, perhaps standardizing behind one makes sense.

Reply:
Here we're dropping the log prior to the action, so I use present tense.  In other places we drop the log after.

I mostly prefer to log after for debugging synchronization problems, but here there are two different return paths, and I got a little lazy and put it before.  I'll just log both paths and use past tense.